notebooks/query-language-object-based-dialect/01.01.quantifier_capabilities.ipynb

```diff
@@ -0,0 +1,842 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b0f5e933",
+   "metadata": {},
+   "source": [
+    "# Hatchet Query Language\n",
+    "\n",
+    "\n",
+    "This notebook explores [Hatchet](https://github.com/LLNL/hatchet) queries from it's [**Object-based Dialect**](https://hatchet.readthedocs.io/en/latest/query_lang.html) within **Category 1: Quantifier Capabilities**. Below, Fig. 1 provides an overview of all the categories within the Hatcher Query Language.   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f1f3163",
+   "metadata": {},
+   "source": [
+    "|Category ID|Category Description|\n",
+    "|:---------:|:-------------------|\n",
+    "|1          |Quantifier Capabilities|\n",
+    "|2          |String Equivalence and Regex Matching|\n",
+    "|3          |String Containment (contains, starts with, ends with)|\n",
+    "|4          |Basic Numeric Comparison (==, >, >=, etc.)|\n",
+    "|5          |Comparison with Special Values (NaN, Inf, None)|\n",
+    "\n",
+    "**Figure 1**: A table of the Hatchet Query Language capabilities, distinguished into categories and their corresponding cateogry ID."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb6dbe40",
+   "metadata": {},
+   "source": [
+    "The notebook covers different query types that are capable of matching various number of nodes from [GraphFrame objects](https://hatchet.readthedocs.io/en/latest/user_guide.html). Hatchet offers multiple interfaces to define queries with different trade-offs to verbosity and expressiveness. The entire capability table with information about all the query use cases, categories and capabilities can be found [here](https://docs.google.com/spreadsheets/d/1fKNlHmDJdDbnE4jyMcaFqdnw6ZSaexgm33rOcVAj0do/edit#gid=0).\n",
+    "\n",
+    "\n",
+    "A hatchet query always finds all **matching paths** for a provided GraphFrame. For this reason, Hatchet queries also takes the shape of a sequence of **predicates** to match, with the added convienience of also allowing to define an accompinying **quantifier** for each predicate. This concept is reflected in Fig. 2, where we interpret queries in terms of filtering or selecting parts of the GraphFrame."
```

Comment:
Consider this rephrase. 

Hatchet query language consumes a [GraphFrame] and a sequence of queries. Each query can comprise a **predicate** and a **quantifier**. Hatchet query language finds all **matching paths** from a provided GraphFrame. For example, in Fig.2,  for the query (any with A or B), the output would comprise of 2 paths, [1, 2, 4] and [1, 3, 4].

Reply:
To add to this, if a rephrase isn't done. I believe takes should be changed to take. Small change, but sentence seems to flow better.